src/preamble.js

```diff
@@ -782,11 +781,11 @@ function getBinaryPromise() {
       }
       return response['arrayBuffer']();
     }).catch(function () {
-      return getBinary();
+      return getBinary(wasmBinaryFile);
     });
   }
   // Otherwise, getBinary should be able to get it synchronously
-  return Promise.resolve().then(getBinary);
+  return Promise.resolve().then(function() { return getBinary(wasmBinaryFile); });
```

Comment:
wasn't this in another PR? looks very familiar..

Reply:
Yep, this PR supersedes #13050, which also had this change.